### PR TITLE
SISRP-52450: Add hot deploy scripts

### PR DESCRIPTION
**Pull request by developish**
https://jira-secure.berkeley.edu/browse/SISRP-52450

### CODE REVIEW

File: script/update-build-tomcat.sh [hunk 1]
```diff
@@ -7,66 +7,106 @@
...
+     echo "WAR_URL variable not defined in the node"| ${LOGIT}
+     exit 1
+fi
+
 # Update source tree (from which these scripts run)
 
 # Change dir to the calcentral home dir to update the source code from GitHub
 cd ${CLC_HOME} || exit 1
-./script/update-source.sh
+#####./script/update-source.sh
```
Comment (developish) [inline script/update-build-tomcat.sh:73]: Does this line need to fully removed from this script?

File: script/update-build-tomcat.sh [hunk 2]
```diff
@@ -119,4 +152,4 @@ chmod -R g+w *
 # Give execute to Others for directories only
 find /var/www/html/calcentral/ -type d -execdir chmod o+x {} \;
 
-exit 0
+###exit 0
```
Comment (developish) [inline script/update-build-tomcat.sh:155]: Should the line be removed?

File: script/update-build-tomcat.sh (hunk 1 above)
Comment (SusanaParker) [reply script/update-build-tomcat.sh:73]: Yes, the line can be removed. I left it while we were testing.

File: script/update-build-tomcat.sh (hunk 2 above)
Comment (SusanaParker) [reply script/update-build-tomcat.sh:155]: Yes, you can remove that line. I will revisit when better error handling is incorporated to the script. For now it is just a modified version of the existing one, and will suffice.

**Comment by developish** [comment]
Susana wrote this code. I just assisted with the git process and reviewed it. 👍

**Pull request merged**